handbook/FileService-MinIO/FileService-MinIO.md

```diff
@@ -0,0 +1,59 @@
+# Leverage Cloud File Storage with MinIO
+
+SystemLink File Service allows you to configure and use the Amazon S3 cloud storage instead of a file share. While NI officially addresses the Amazon S3 cloud storage, the File Service also works with MinIO, a server-side software storage stack that is compatible with Amazon S3.
+
+To use MinIO with SystemLink, you need SystemLink version 2020 R3 or later.
+
+To use MinIO as a storage provider, set up the MinIO server on a system that you would like to upload files to. This can be the same machine as the SystemLink server or an own dedicated server. Then, configure the File Service to use that server.
+
+## Running MinIO server
+
+1. Download the server application from the [MinIO website](https://min.io/download).
+
+2. From the command line, run the following command to run the server:
+
+    ```bash
+    minio server <PathToRootFolder>
+    ```
+
+    <figure>
+    <img src="../../img/minio-server.png" width="500" />
+    <figcaption>Running a minio server.</figcaption>
+    </figure>
+
+3. When starting the MinIO server for the first time, use a web browser to connect to the endpoint that the command line prints out. Create a bucket by using the `+` button in the right bottom corner.
+
+    !!! note "Note"
+        - It is recommended to set non-default access key and secret key. You can do that by setting the environment variables `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`. Run the following two commands from the command line just before starting the server.
+        ```
+        set MINIO_ACCESS_KEY=YourAccessKey
```

Comment:
For deployment, I assume the user will want a .bat file or powershell script to launch the server with the appropriate configuration. Should we provide example source for such a file here, allowing the user to easily copy-paste what they need to modify and run on their system?

Reply:
Are you asking for a script file that adds the two required settings "S3BackEndServiceUrl" and "S3ForcePathStyle" to the FileIngestion json file or what is it you're asking for?

At any rate, an example for the json content is already given at the very bottom which they can copy to the file.

Reply:
No, I am suggesting a script for running the service with the correct environment. Or to just gather the entire command line sequence in one clear code block, as is common practice for these sort of instructions.

Reply:
Okay, just amended the two code lines to a full example.
![image](https://user-images.githubusercontent.com/42576216/114224086-e8706900-9970-11eb-8aff-5a9ffce78094.png)

Reply:
I think that writing a batch file would be overkill, since you had to pass in three arguments, make sure that there are three arguments, make sure that the path is valid, print out a meaningful help.... That's a lot for three simple lines of code.